compiler/src/iree/compiler/Codegen/Common/CPU/test/llvmcpu_materialize_encoding.mlir

```diff
@@ -1298,6 +1298,200 @@ func.func @matmul_lowering_i8i8i32_aarch64_i8mm() attributes {
 
 // -----
 
+#map = affine_map<(d0, d1, d2) -> (d0, d2)>
+#map1 = affine_map<(d0, d1, d2) -> (d2, d1)>
+#map2 = affine_map<(d0, d1, d2) -> (d0, d1)>
+func.func @matmul_lowering_i8i4i32_aarch64() attributes {
+  hal.executable.target = #hal.executable.target<"xyz", "xyz", {target_triple="aarch64-xyz-xyz"}>
```

Comment:
Is the intent to not have `ukernels = "all"` for this tests?

Reply:
Correct, this tests the path where ukernels are not enabled so it'll fall into the `if (!hasUkernel(target)) { ... }` block. This checks that it'll use the 4x16x1 tile size inside of that if statement.